ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -0,0 +1,61 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# prod
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_values_axis(available_dtypes=ivy.current_backend().valid_dtypes),
+    dtype=st.sampled_from(ivy.current_backend().valid_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.prod"
+    ),
+    keepdims=st.booleans(),
+    initial=st.one_of(st.booleans(), st.integers(), st.floats(), st.complex_numbers())
+)
+def test_numpy_prod(
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+    keepdims,
+    initial
+):
+    input_dtype, x, axis = dtype_and_x
```

Comment:
I just found out, the output format of helpers.dtype_values_axis when requesting multiple arrays will be (dtype, [arrays, arrays, ...]). So this line here should be:
 input_dtype, [x, axis] = dtype_and_x

Reply:
`input_dtype, [x, axis] = dtype_and_x` results in `ValueError: too many values to unpack (expected 2)`

Reply:
Oh, never mind, I made a mistake. Sorry.